### ISSUE TITLE
Using API is 30 minutes delayed data

### ISSUE THREAD

**Issue by sumit2k21** [bombardier-gif/covid-vaccine-booking#118]
https://apisetu.gov.in/public/marketplace/api/cowin/cowin-public-v2
Here we can see they are saying if we are using APIs, the availability data would be 30 minutes delayed, it makes it not much useful.
Are we using Co-WIN Protected APIs? seems like we are using protected APIs. so Is the data real time ? 
anything can be done in this regard ?

**Comment by sumit2k21** [comment]
Also can you go to insights->traffic and update us how many clone this repository has ?

**Comment by bombardier-gif** [comment]
TLDR: People are daily able to book slots using this script. Yes, data might be stale but nothing much to do here. 
What i am interested is that the arogya setu app needs no captcha so if we figure out those APIs?

**Comment by thepitzaboy1** [comment]
You can use fiddler on IOS to capture the API calls?

**Comment by vesan83** [comment]
> TLDR: People are daily able to book slots using this script. Yes, data might be stale but nothing much to do here.
> What i am interested is that the arogya setu app needs no captcha so if we figure out those APIs?

Are people in places like Mumbai able to book the appointment? i just saw online availablaity at a hospital, but the script running on my laptop was not even able to detect it. is it due to the 30 min delay ?
even the telegram channel of under45.in detected it

**Comment by vesan83** [comment]
i can confirm there is no delay, as soon as the slot opens on cowin website, we are getting trigger using the script

**Comment by xxxAng** [comment]
@vesan83 what is the polling interval you have used ? 1 sec, 5 sec or 15 sec?

**Comment by bombardier-gif** [comment]
getting trigger is a hit or miss to be honest. there is one possible bug in the filtering which could be the root cause of missing slots

**Comment by vesan83** [comment]
@xxxAng i have used it as 15sec

@bombardier-gif since there is a bug, is it better to use pincode search ?
